staging/src/k8s.io/apiserver/pkg/endpoints/handlers/delete.go

```diff
@@ -214,13 +215,14 @@ func DeleteCollection(r rest.CollectionDeleter, checkBody bool, scope *RequestSc
 
 		options := &metav1.DeleteOptions{}
 		if checkBody {
-			body, err := limitedReadBody(req, scope.MaxRequestBodyBytes)
+			body, err := limitedReadBodyWithRecordMetric(ctx, req, scope.MaxRequestBodyBytes, scope.Resource.GroupResource().String(), requestmetrics.Delete)
```

Comment:
DELETECOLLECTION is its own verb and probably should not use requestmetrics.Delete here

Reply:
Fixed in latest commit, PTAL~